packages/protocol/scripts/truffle/deploy_release_contracts.ts

```diff
@@ -139,6 +155,34 @@ async function handleJSONFile(err, data) {
     return
   }
 
+  console.info('Verifying grants have not already been deployed.')
+  if (!argv.yesreally) {
+    for (const grant of grants) {
+      if (deployedGrants.includes(grant.identifier)) {
+        console.info(
+          chalk.red(
+            'Grant with identifier ' + grant.identifier + ' has already been deployed.\nExiting.'
+          )
+        )
+        process.exit(0)
+      }
+      // Sum occurences of each identifier in the grant file, if more than 1 then there is a duplicate.
+      const identifierCounts = grants.map((x) => (x.identifier === grant.identifier ? 1 : 0))
```

Comment:
You can use `grants.some` here

Reply:
I prefer my solution since it catches duplicate identifiers and duplicate grants, I could only formulate a `some` that would only catch duplicate identifiers, i.e. find a grant in this list that isn't exactly the current grant.

Reply:
I don't see how what you're doing here is any different than using `grants.some` to check for duplicate identifiers